Marlin/Configuration_adv.h

```diff
@@ -2896,13 +2896,17 @@
  */
 //#define SPINDLE_FEATURE
 //#define LASER_FEATURE
+
+
 #if EITHER(SPINDLE_FEATURE, LASER_FEATURE)
   #define SPINDLE_LASER_ACTIVE_STATE    LOW    // Set to "HIGH" if the on/off function is active HIGH
   #define SPINDLE_LASER_PWM             true   // Set to "true" if your controller supports setting the speed/power
   #define SPINDLE_LASER_PWM_INVERT      false  // Set to "true" if the speed/power goes up when you want it to go slower
 
   #define SPINDLE_LASER_FREQUENCY       2500   // (Hz) Spindle/laser frequency (only on supported HALs: AVR and LPC)
 
+
+  //#define SPINDLE_SERVO // If you are using RC ESC to drive spindle motor. Remember to enable servos and disable SPINDLE_LASER_FREQUENCY
```

Comment:
You will want a blank line between your new setting and the `CUTTER_POWER_UNIT` section.